nicolette-LewNic1/jquery-geoquakes-lab/scripts/app.js

```diff
@@ -0,0 +1,110 @@
+/*Part 1. Rendering Data
+-Data structure: HTTP/REST
+-Lists 1 earthquake 
+
+-having difficulty with map
+*/
+
+var quakes = "https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/4.5_week.geojson";
+
+$(document).ready(function() {
+  console.log("Let's get coding!");
+  let map = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 37.78, lng: -122.44},
+	zoom: 3
+  };
+
+  $.ajax({
+	method: 'GET',
+	url: quakes,
+	success: function(response) {
+		console.log(success);
+		var title = response.metadata.title;
+		console.log(response.metadata.title);
```

Comment:
I like the logs checking what you are saving, but remember to log the variable itself instead to make sure the data is being stored correctly.